rust/repositories.bzl

```diff
@@ -586,31 +586,35 @@ def rust_repository_set(
     """Assembles a remote repository for the given toolchain params, produces a proxy repository \
     to contain the toolchain declaration, and registers the toolchains.
 
-    N.B. A "proxy repository" is needed to allow for registering the toolchain (with constraints) \
-    without actually downloading the toolchain.
-
     Args:
         name (str): The name of the generated repository
         version (str): The version of the tool among "nightly", "beta', or an exact version.
         exec_triple (str): The Rust-style target that this compiler runs on
-        include_rustc_srcs (bool, optional): Whether to download rustc's src code. This is required in order to use rust-analyzer support. Defaults to False.
-        allocator_library (str, optional): Target that provides allocator functions when rust_library targets are embedded in a cc_binary.
+        include_rustc_srcs (bool, optional): **Deprecated** - instead see [rust_analyzer_toolchain_repository](#rust_analyzer_toolchain_repository).
+        allocator_library (str, optional): Target that provides allocator functions when rust_library targets are
+            embedded in a cc_binary.
         extra_target_triples (list, optional): Additional rust-style targets that this set of
-            toolchains should support. Defaults to [].
```

Comment:
why drop these defaults?

Reply:
The text here is redundant. Defaults are neatly rendered in our docs

Reply:
https://bazelbuild.github.io/rules_rust/flatten.html#rust_repository_set

Reply:
Thanks!